samples/AzCopyCore/AzCopyCore/AzCopyCore.csproj

```diff
@@ -19,18 +21,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26223-02" />
-    <PackageReference Include="System.Memory" Version="$(CoreFxPackageVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(CoreFxPackageVersion)" />
+    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="1.0.0-alpha-26320-02" />
+    <PackageReference Include="System.Buffers.ReaderWriter" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Memory" Version="4.5.0-preview3-26319-04" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.0-preview3-26319-04" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Azure.Experimental" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Buffers.Experimental" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Text.Http" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Text.Http.Parser" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.Text.Utf8String" Version="$(CoreFxLabPackageVersion)" />
-    <PackageReference Include="System.IO.Pipelines" Version="$(CoreFxLabPackageVersion)" />
+    <PackageReference Include="System.Azure.Experimental" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Buffers.Experimental" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Text.Http" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Text.Http.Parser" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.Text.Utf8String" Version="0.1.0-preview2-180320-2" />
+    <PackageReference Include="System.IO.Pipelines" Version="4.5.0-preview3-26319-04" />
```

Comment:
nit: Move this up to the other ItemGroup with the rest of the CoreFX packages

Reply:
will do